### DISCUSSION TITLE
ADMOB REAL AND TEST (ANDROID)

### DISCUSSION THREAD

**Discussion by alexisbogarin** [Poing-Studios/Godot-AdMob-Android-iOS#77] [Q&A]
Hi! I've been setting up the plugins and the test ads work great. But putting everything to work with real ads keeps getting test ads.

![1](https://user-images.githubusercontent.com/51892359/121831292-79c7c880-cc9d-11eb-95ba-9d2d73b49184.jpg)
![2](https://user-images.githubusercontent.com/51892359/121831296-7af8f580-cc9d-11eb-8ffb-f442451534e0.jpg)

What could be the problem or the reason?

**Comment by gumaciel** [comment]
As I know, the "Test Ads" will only disappear when you add your app on AdMob.com, but I'm not sure, I will give a look at night and give you a feedback.

**Reply by alexisbogarin** [reply]
![Sin título](https://user-images.githubusercontent.com/51892359/121967910-9d474d80-cd47-11eb-9411-1ca962fd9dd6.png)
More info:
This appears to me in ADMOB. I shared my game with another phone to test it and it still shows test ads. 
But my admob parses it as real?
Everything is very confusing

**Reply by gumaciel** [reply]
Are you using the ads by the app downloaded on Google Play?

**Reply by gumaciel** [reply]
- If you mark "is real", so the app should appear real ads when goes to production (and also when is not on production, but if isn't on production I guess will appear a Label like: "Test Ads")

**Reply by alexisbogarin** [reply]
I don't have the play store. Look for information and they told me that it may be the version when exporting. I have is real in true and I still get test ads. I will keep trying, thank you very much for answering! :)

**Reply by mani144** [reply]
@gumaciel is  ( is real ) option still exists in the new build of the plugin, i cant find it anymore.

**Reply by gumaciel** [reply]
hi @mani144 , it was removed, now it's using release tags `OS.has_feature("release")`:

https://github.com/Poing-Studios/godot-admob-editor/blob/master/addons/admob/src/singletons/AdMobSingleton.gd#L57

what means: if is on "release", then will set `is_real = true` 

this change was made to avoid bugs, in case the user sends a version to production with `is_real = false`

read more: 
https://github.com/Poing-Studios/godot-admob-editor/issues/16
https://github.com/Poing-Studios/godot-admob-editor/issues/13
https://github.com/Poing-Studios/godot-admob-editor/issues/6
